styles/components/_bankBoard.scss

```diff
@@ -1,28 +1,29 @@
 .bank {
```

Comment:
since bank as bank is not used, how about you call everything without the '__board' part? So main div (bankView) would have a class of bank, then we would have bank__count, bank__image etc?